pkg/config/config.go

```diff
@@ -16,17 +15,16 @@ type Configs struct {
 	Tasks map[string][]Task
 }
 
-func GetConfigs() Configs {
-	// TODO Should get the name of the Dunner file from a Constant
-	fileContents, err := ioutil.ReadFile("./.dunner.yaml")
+func GetConfigs(filename string) (*Configs, error) {
+	fileContents, err := ioutil.ReadFile(filename)
 	if err != nil {
-		log.Fatal(err)
+		return nil, err
```

Comment:
Don't you think having the `log.Fatal(err)` would also be helpful?

Reply:
Actually, it would be better if we return the error and handle in `dunner.go`. There (and everywhere else too), we could replace `panic` with `log.fatal`. I was going to open a separate issue as improvement to replace `panic` with either `log.fatal` or anything else if third-party library is used.

Reply:
Okay, I checked again. So I have to log the error if it is of different package/module, and return those errors which are due to execution flow of the function. Logging between the modules is acceptable.
I will resolve it soon.